src/customers/customers-all.md

```diff
@@ -18,3 +18,52 @@ _All Customers_{:.b2b-only}
    To learn more, see [Updating Customer Accounts]({% link customers/customer-account-update.md %}).
 
 1. When complete, click <span class="btn">Save Customer</span>.
+
+## Workspace Controls
+
+|Control|Description|
+|--- |--- |
+|Add New Customer|Creates a new customer account.|
+|Search|Initiates a search for customers based on the current filters.|
+|Filters|Defines a set of search parameters used to filter the records that appear in the [grid]({% link stores/admin-grid-controls.md %}).|
+|[Default View]({% link stores/admin-grid-layout.md %})|Determines the default column layout of the grid.|
+|Columns|Determines the selection of [columns]({% link stores/admin-grid-layout.md %}) and their accounts in the grid. The column layout can be changed and saved as a _view_. By default, only some of the columns are included in the grid.|
+|Export|Exports the selected records as a CSV or Excel XML file.|
+
+## Column descriptions
```

Comment:
From what I can tell the **Column descriptions** and **Additional Columns Available** match fairly closely to the following existing tables:
- https://docs.magento.com/user-guide/customers/customer-account-manage.html#columns
- https://docs.magento.com/user-guide/customers/customer-account-manage.html#additional-columns
Please make sure the descriptions in your PR match those in the above two links.

Reply:
Hi @erikmarr, thank you for reviewing!

I have implemented your suggestions, could you please check and approve?